Makefile

```diff
@@ -420,6 +428,12 @@ $(appliance): isos/appliance.sh isos/appliance/* isos/vicadmin/** $(vicadmin) $(
 	@echo building VCH appliance ISO
 	@$(TIME) $< -p $(appliance-staging) -b $(BIN)
 
+# main appliance target + kubelet - depends on all top level component targets
+$(appliance-vkube): isos/appliance.sh isos/appliance/* isos/vicadmin/** $(vicadmin) $(vic-init) $(portlayerapi) $(docker-engine-api) $(appliance-staging) $(archive)
+	@echo building VCH appliance ISO
+	@$(TIME) $< -p $(appliance-staging) -b $(BIN) -x $(VIRTUAL_KUBELET) -o appliance.vkube
```

Comment:
Use `$@` instead of `appliance.vkube` - that way you'll actually get conditional builds if tweaking names.